runtime/compiler/env/VMJ9Server.cpp

```diff
@@ -2616,6 +2616,65 @@ TR_J9ServerVM::isOffHeapAllocationEnabled()
    return vmInfo->_isOffHeapAllocationEnabled;
    }
 
+
+TR_arrayTypeCode
+TR_J9ServerVM::getPrimitiveArrayTypeCode(TR_OpaqueClassBlock* clazz)
+   {
+   TR_ASSERT(isPrimitiveClass(clazz), "Expect primitive class in TR_J9VMBase::getPrimitiveArrayType");
```

Comment:
Please change the message to:
"Expect primitive class in TR_J9ServerVM::getPrimitiveArrayType"